R-package/R/lgb.model.dt.tree.R

```diff
@@ -111,7 +111,12 @@ single.tree.parse <- function(lgb_tree) {
         , leaf_count = integer(0L)
       )
       # start tree traversal
-      pre_order_traversal(env, tree_node_leaf, current_depth, parent_index)
+      pre_order_traversal(
+        env = env
+        , tree_node_leaf = tree_node_leaf
+        , current_depth = current_depth
+        , parent_index = parent_index
+        )
```

Comment:
can you please align this `)` with the `p` in `pre`?

```suggestion
      )
```